packages/react-components/src/identifiers/IdentifierProvider.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This diff looks wrong to me. Perhaps you wanted to create the PR against another branch?

Reply:
Nope its correct. The file is named incorrectly. `IndentifierProvider`
Renaming the file. That's why you see it here like this.

Reply:
Renamed the file. That's why you see it like this.

Reply:
Hmm. Github could do a better job with tracking renames then...